features/issues/components/issue-list/issue-row.tsx

```diff
@@ -21,7 +21,7 @@ export function IssueRow({ projectLanguage, issue }: IssueRowProps) {
   const firstLineOfStackTrace = stack.split("\n")[1];
 
   return (
-    <tr className={styles.row}>
+    <tr className={styles.row} data-testid={`issue-${issue.id}`}>
```

Comment:
I'm not sure if we need to prefix "issue" to these odds ids.

uuids are theoretically unique app-wide and while testing I don't think we'll have any overlap of context when looking for test ids. But maybe. Hrm.

Reply:
You know, I've thought about this and like, yeah. No reason to change this, it's good and probably a better habit. I was just thinking that, like, a lot of apps, even those that are pretty mature and long in the tooth, use basic auto-incremented numerical ids--and for many good reasons.

So, disregard my knee jerk reaction!